meadow-client/src/MainFrame.purs

```diff
@@ -435,9 +437,9 @@ evalF (CompileProgram next) = do
       pure next
 
 evalF (SendResult next) = do
-  mContract <- use (_runResult)
+  mContract <- use (_compilationResult)
   let contract = case mContract of
-                   Success (Right (RunResult x)) -> x
+                   Success (Right x) -> view (_InterpreterResult <<< to (\r -> r.result) <<< _RunResult) x
```

Comment:
Make a `prop` lens. You'll be happier.

Reply:
`prop` lenses are _very_ cool

Reply:
Type-level programming at low, low prices. :-D